[2024-06-11 10:50:13,286][backoff][INFO] - Backing off chat_completion(...) for 0.0s (aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:26,058][backoff][INFO] - Backing off chat_completion(...) for 0.7s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:27,365][backoff][INFO] - Backing off chat_completion(...) for 0.3s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:28,230][backoff][INFO] - Backing off chat_completion(...) for 0.2s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:29,100][backoff][INFO] - Backing off chat_completion(...) for 4.0s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:33,682][backoff][INFO] - Backing off chat_completion(...) for 5.2s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:10:39,391][backoff][INFO] - Backing off chat_completion(...) for 22.3s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:11:02,372][backoff][INFO] - Backing off chat_completion(...) for 40.5s (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-11 12:11:43,460][backoff][ERROR] - Giving up chat_completion(...) after 8 tries (aiohttp.client_exceptions.ClientResponseError: 429, message='Too Many Requests', url=URL('https://api.anthropic.com/v1/messages'))
